contracts/Vault.vy

```diff
@@ -8,22 +8,38 @@ from vyper.interfaces import ERC20
 
 implements: ERC20
 
+
 interface DetailedERC20:
-    def name() -> String[42]: view
-    def symbol() -> String[20]: view
-    def decimals() -> uint256: view
```

Comment:
Can `blackadder` be adjusted to maintain this? I find this must more readable actually (the item after the `:` is actually the mutability of the method, not a function body)

Reply:
I guess we'd have to patch black's `transform_line` or the `Line` class. It's doable, but will require some investigation/time

> I find this much more readable

agree. And aesthetic

Reply:
fyi I probably won't be able to look into this until next week

Reply:
What's weird about this is that `Strategy` interface below didn't get update... :thinking:

Reply:
Not sure I get what you mean, I see `Strategy` reformatted below here

Reply:
Weird, I don't
```
interface Strategy:
    def distributeRewards(_shares: uint256): nonpayable
    def estimatedTotalAssets() -> uint256: view
    def withdraw(_amount: uint256): nonpayable
    def migrate(_newStrategy: address): nonpayable
```

Reply:
It's in my original commit https://github.com/iearn-finance/yearn-vaults/pull/52/commits/9e661292314d4ca2c7f34238b4415eef1209fe41. Looks like you reverted that here https://github.com/iearn-finance/yearn-vaults/pull/52/commits/565b7370c4bf0c4db02a6f4d68bc471e5ebb51f3

Reply:
ah, sorry!

Reply:
@fubuloubu for now, I'd suggest to keep this formatting as is, if it's not too much of an inconvenience. We should address these formatting issues in the next major blackadder release but it will take some time until we get that out.
Otherwise, we can postpone this PR until then.

Reply:
I'd argue we should wait then. This reformatting makes things significantly less clear.